qubes_config/global_config.glade

```diff
@@ -6329,7 +6234,7 @@ Documentation will open in a disposable qube.</property>
                           <object class="GtkLabel">
                             <property name="visible">True</property>
                             <property name="can-focus">False</property>
-                            <property name="label" translatable="yes">This policy governs Qubes OS behavior when using &lt;tt&gt;qubes-open-in-dvm&lt;/tt&gt; with URLs.</property>
```

Comment:
I think `qubes-` instead of `qvm-` was a typo, but if `qvm-open-in-[d]vm` is really changing to `qubes-open-in-[d]vm` in Qubes 4.2, let me know.

Reply:
(Confirmed to be just a typo via chat.)